Changelog.md

```diff
@@ -1,3 +1,14 @@
+## 2.6.0 - Unreleased
+
+### Changed
+
+-   Use of `TEST_SETUP` command instead of `TRANSMITTER_TEST` to set TX value.
```

Comment:
This looks like implementation detail and not something user is facing.. important enough to mention?

Reply:
They is interacting with it when they have a custom firmware that needs to handle the commands.
I mean it is a spec, and the device is expected to have the whole spec supported, but I'm thinking it may be relevant for people with custom firmware (can't tell you how much).

But the change is also fair, let me know if I should still change it like that

Reply:
Yes, please.